Source/SwiftLintFramework/Extensions/Configuration+Parsing.swift

```diff
@@ -10,7 +10,7 @@ extension Configuration {
         case swiftlintVersion = "swiftlint_version"
         case useNestedConfigs = "use_nested_configs" // deprecated
         case warningThreshold = "warning_threshold"
-        case whitelistRules = "whitelist_rules"
+        case allowlistRules = "allowlist_rules"
```

Comment:
can we keep parsing `whitelist_rules` and print a warning saying it's deprecated? I don't think it's fair to break the setup for everyone using it (https://github.com/realm/SwiftLint/issues/3274)

Reply:
Also this should be mentioned in the CHANGELOG